buildshiprun/handler.go

```diff
@@ -453,3 +451,17 @@ func getRegistryAuthSecret() string {
 	}
 	return ""
 }
+
+func getMemoryLimit() string {
+	suffix := "m"
+	memoryLimit := os.Getenv("default_memory_limit")
+	if _, present := os.LookupEnv("KUBERNETES_SERVICE_PORT"); present == true {
```

Comment:
I'd like to see this split out into a separate method which is also tested independently. In the future we may want to change the way we find out whether we're in Kubernetes and so having this abstracted would be ideal.

No need to move to the sdk yet though.